### ISSUE TITLE
Ticking Block Entity reappears on server start

### ISSUE THREAD

**Issue by DeviousRex** [raoulvdberge/refinedstorage#1217]
I've been experiencing several different ticking block entity issues over the past several months. They are:

* Sometimes I'm able to remove the offending entity from refinedstorage_nodes.dat and the server starts fine. 

* Sometimes I have to remove the entity from refinedstorage_nodes.dat and then again from the affected region (r.0.0.mca) before the server will successfully start. 

* In rare cases, such as this one,  The offending entity is removed from refinedstorage_nodes.dat and from the region, but when I start the server again, the server crashes with the same ticking block entity and when I look in the region file, the entity is back. When I check refinedstorage_nodes.dat, it is NOT in that file.

In this case, I also removed the Refined Storage entities that were connected to the ticking block. When I restarted the server, it crashed all the entity entries I removed from r.0.0.mca were back.

I expect that when I remove the ticking entity from refinedstorage_nodes.dat that the entity will be removed and the server will start.

What is happening is that the entity is somehow added back and it crashes the server.

Steps to reproduce:
1. Server Crashes and creates a log
2. Review the log and see that the issue is a ticking block entity. (fluid_interface this time, although I have also had the problem with an external_storage entity, grid, and a couple of others)
3. Open refinedstorage_nodes.dat and remove the offending entity.
4. Open r.0.0.mca and remove the offending entity
5. Remove entities directly connected to the offending entity.
6. Save both files and ensure they are updated on the server.
7. Restart server
8. Server Crashes 
9. Repeat step 2.
10. Open r.0.0.mca and notice all the previously removed entities are back.
11. Open refinedstorage_nodes.dat and notice removed entities are still removed.

#### Version (Make sure you are on the latest version before reporting):

- Minecraft: 1.11.2
- Forge:13.20.0.2282
- Refined Storage:1.4.4

Does this issue occur on a server? [yes/no]
Yes

#### If a (crash)log is relevant for this issue, link it here:
[crash-2017-05-13_13.03.08-server.txt](https://github.com/raoulvdberge/refinedstorage/files/998927/crash-2017-05-13_13.03.08-server.txt)

All of these entities are running in the Spawn Chunks. Could this be causing an issue?

Is there something saved locally that would cause entities to be replaced in the world after they have been removed from the server dat and mca files?

Thanks for your help.

**Comment by raoulvdberge** [comment]
Hrm, I think I fixed this for 1.4.5 with my refactors I've done.

**Comment by raoulvdberge** [comment]
Problem:

RS thinks that on the position of a fluid interface there is a cable.

**Comment by DeviousRex** [comment]
Thank you! When do you estimate 1.4.5 will be available?

Do you have a recommendation how to fix it in the meantime?

Do you have any idea why, when I remove entities from the dat and msa file,
they reappear when I restart the server? Is there an artifact entry
somewhere?

Thanks again for all your help. I have another issue regarding controllers
turning off, causing upgrades to disappear from devices, but I want to do
some more testing to see exactly what the steps are. I know there were two
issues logged about this before, but you mentioned they were fixed in
earlier versions and I am still experiencing it.

Cheers,

Dennis

**Comment by raoulvdberge** [comment]
1.4.5 got quite a few changes, and still needs some polish.

Furthermore, I also fixed the "stuff disappearing from machines" bug (I think). It has to be a tested a bit more, so it'll delay the update a bit.

For a temporary fix, the machine would be coming back because you didn't properly remove the tile entity or network node.

You have to:
- Remove the tile entity from the chunk data
- Remove the network node from refinedstorage_nodes.dat

**Comment by DeviousRex** [comment]
Thanks.

I definitely removed the fluid_interface from both files... and then I
removed all of the other network nodes (controller, disk drive, crafting
grid, fluid grid, cables) from the chunk data... I did not remove all the
collateral nodes from the dat file... and I did not remove the cables...
mostly because I can't figure out a good way to identify individual nodes
among many in the dat file. In the chunk data there are x,y,z coords, but
only the longer number in the dat file. How do I read the position number
in the dat file to figure out where it is?

Thank you again for all of your help. Your mod has changed the game for me
and I really love and appreciate it and the time you put into it. I also
appreciate you taking your time to answer my questions.

Regards,

Dennis
